websites/E/EternalNetwork TM/presence.ts

```diff
@@ -138,16 +138,16 @@ presence.on("UpdateData", async () => {
 
 			if (pathname.startsWith("/")) presenceData.state = ttl;
 
-			if (hash.includes("page_ABOUT")) presenceData.state = "About info page !";
+			if (hash.includes("page_ABOUT")) presenceData.state = "About info page";
 
 			if (hash.includes("page_PROGRAMS"))
-				presenceData.state = "Checking radio program !";
+				presenceData.state = "Checking radio program";
 
 			if (hash.includes("page_REQUEST"))
-				presenceData.state = "Requesting a song !";
+				presenceData.state = "Requesting a song";
 
 			if (hash.includes("page_CONTACTS"))
-				presenceData.state = "Contact us page !";
+				presenceData.state = "Contact us page";
```

Comment:
Viewing? Checking? You need to be consistent and have a verb here too

Reply:
Message changed.